caffe2/serialize/inline_container.cc

```diff
@@ -371,7 +372,11 @@ std::tuple<at::DataPtr, size_t> PyTorchStreamReader::getRecord(
   mz_zip_archive_file_stat stat;
   mz_zip_reader_file_stat(ar_.get(), key, &stat);
   valid("retrieving file meta-data for ", name.c_str());
-  at::DataPtr retval = c10::GetCPUAllocator()->allocate(stat.m_uncomp_size);
+  at::Allocator* allocatorPtr = c10::GetCPUAllocator();
+  if (allocator.has_value()) {
+    allocatorPtr = allocator.value();
+  }
```

Comment:
```suggestion
  at::Allocator* allocatorPtr = allocator.has_value() ? allocator.value() : c10::GetCPUAllocator();
```

To avoid the extra getcpuallocator call.
Same below